src/cf-icons/src/cf-icons.less

```diff
@@ -396,12 +397,14 @@
 .cf-icon__rotate( @degrees, @rotation ) {
     // TOOD: Check whether `filter` can be replaced
     //       by `-ms-transform` for CF supported browsers.
-    filter: progid:DXImageTransform.Microsoft.BasicImage( rotation=@rotation );
+    /* stylelint-disable-next-line function-name-case */
+    filter: progid:dximagetransform.microsoft.basicimage( rotation=@rotation );
```

Comment:
Did you mean to revert the auto-fix on the function name here and a few lines below?

Another approach (besides ignoring these lines) would be using the ignore option for this rule: https://stylelint.io/user-guide/rules/function-name-case/#ignorefunctions-regex-as-string-regex-non-regex

Reply:
Oops, nope, fixed. Since this has a TODO to check replacing these filters I'll leave it as inline ignore code comments, since the whole block can likely be stripped out.

Reply:
Hmmmm okay discovered the stylelint ignore comments don't work here either :(
I've updated to use `function-name-case` as you suggested.